From 16ec881da25e90310b0bdfb5a6efbea74a1aa6f2 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Wed, 16 Feb 2022 00:20:13 +0800
Subject: [PATCH 21/50] Revert "amdgpu/pm: Make sysfs pm attributes as
 read-only for VFs"

This reverts commit 7b6dc07c6e690e55a3ddce9588e2da9dd9085fb4.
---
 drivers/gpu/drm/amd/pm/amdgpu_pm.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/gpu/drm/amd/pm/amdgpu_pm.c b/drivers/gpu/drm/amd/pm/amdgpu_pm.c
index 49109614510b..9f383b9041d2 100644
--- a/drivers/gpu/drm/amd/pm/amdgpu_pm.c
+++ b/drivers/gpu/drm/amd/pm/amdgpu_pm.c
@@ -2098,12 +2098,6 @@ static int default_attr_update(struct amdgpu_device *adev, struct amdgpu_device_
 		}
 	}
 
-	/* setting should not be allowed from VF */
-	if (amdgpu_sriov_vf(adev)) {
-		dev_attr->attr.mode &= ~S_IWUGO;
-		dev_attr->store = NULL;
-	}
-
 #undef DEVICE_ATTR_IS
 
 	return 0;
-- 
2.27.0

